api/dataset_test.go

```diff
@@ -479,11 +479,13 @@ func getValidOptionsResponse() dataset.Options {
 func getValidGeoResponse() *cantabular.GetGeographyDimensionsResponse {
 	return &cantabular.GetGeographyDimensionsResponse{
 		Dataset: gql.Dataset{
-			Variables: gql.Variables{
-				Edges: []gql.Edge{
-					{
-						Node: gql.Node{
-							Name: "NodeName " + uuid.NewString(),
+			RuleBase: gql.RuleBase{
+				IsSourceOf: gql.Variables{
+					Edges: []gql.Edge{
+						{
+							Node: gql.Node{
+								Name: "NodeName " + uuid.NewString(),
+							},
```

Comment:
Why are we changing this?
Didn't this got changed a few days ago?

Reply:
When I upgraded to the latest dp-api-clients-go version, vscode was showing errors so I needed to update these to prevent build failure